compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -953,6 +953,14 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
       }
       break;
 
+    case luci::CircleOpcode::SPLIT:
+      // Only the second input is quantized
```

Comment:
`CircleSplit` input is second( arg(1) ) one and the first one is `split_dim` from schema definition... If you want to reflect this in the comment, how about `Input is second argument for 'SPLIT' from schema.` ? Or if this is not the case, could you explain a bit more about this comment?

Reply:
@seanshpark  You're right about my intention to quantize the second argument.

I wrote the comment in an operator-independent way, because there can be another operator that falls within the same category, i.e., only the second input is quantized.

Reply:
I've updated the comment PTAL.